src/reducers/users.tsx

```diff
@@ -49,6 +49,8 @@ export type User = {
 };
 
 export const actions = {
+  abortLoadCurrentUser: createAction('ABORT_LOAD_CURRENT_USER'),
+  beginLoadCurrentUser: createAction('BEGIN_LOAD_CURRENT_USER'),
```

Comment:
The thunk function starts with `fetch` so maybe we could use `beginFetchCurrentUser` and `abortFetchCurrentUser`?

We could also use similar function names for the API and redux functions, by choosing either `xxCurrentUser` or `xxCurrentUserProfile`. Not necessarily here, though.

Reply:
Sure. I like shortening everything to `xxCurrentUser` since the final result is `User`.